### PR TITLE
add privacy document regarding the telemetry we collect

### LINKED ISSUE #544: Extend Telemetry documentation
Perhaps we could copy Botkube's or Testkube's telemetry documentation: 
 - https://botkube.io/privacy/
 - https://kubeshop.github.io/testkube/telemetry/

Currently, our Analytics notice is very limited: https://kubeshop.github.io/kusk-gateway/cli/install-cmd/#environment-variables

### PR DESCRIPTION
**Pull request by kylehodgetts**
This PR closes #544 

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/privacy.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+The telemetry data we use in our metrics is limited to:
+
+ - The number of CLI installations.
+ - The number of unique CLI usages in a day.
+ - The number of installations to a cluster.
+ - The number of unique active cluster installations.
+ - The number of people who disable telemetry.
+ - The number of unique sessions in the UI.
+ - The number of API, StaticRoute and EnvoyFleet creations
```
Comment (mbana) [inline docs/privacy.md:46]: Missing full stop.

File: mkdocs.yml [hunk 2]
```diff
@@ -53,10 +53,10 @@ nav:
     - OpenAPI Extension: reference/extension.md
     - Architecture: reference/architecture.md
     - Dashboard API: reference/openapi.md
-    - Analytics: reference/analytics.md
   - Contributing:
     - Development: 'contributing/development.md'
     - Roadmap: 'contributing/roadmap.md'
+  - Privacy Policy: privacy.md
```
Comment (mbana) [inline mkdocs.yml:59]: What's up the indentation here? Also, small point, can you follow the structure of the surrounding block by having a single quote (') around the file.

**Review by mbana — DISMISSED** [review]
LGTM.

Minor points found. Up to you if you decide to do them or not.

**Review by jasmingacic — APPROVED** [review]

**Review by jasmingacic — APPROVED** [review]
Good to go

**Pull request merged**